### ISSUE TITLE
Cloud Load Balancer commands

### ISSUE THREAD

**Issue by jrperritt** [rackspace/rack#312]

**Comment by rgbkrk** [comment]
Yay for this. Any idea what the CLI interface will look like?

**Comment by rgbkrk** [comment]
One thing that's pretty handy to have is the ability to look up a load balancer [node by IP and Port](https://github.com/rgbkrk/peekaboo/blob/812c376c699020c1214537edb81d277df73e29a8/peekaboo.go#L59-L88).

**Comment by jrperritt** [comment]
Like the others:
`rack load-balancers node create --address 127.0.0.1`
`rack load-balancers load-balancer create --name lb1 --nodes nodeID1,nodeID2 --port 80`

**Comment by jrperritt** [comment]
Finding an LB node by IP and/or port could be optional flags to the `get` subcommand. Of course then `id` would be optional.

**Comment by rgbkrk** [comment]
Something like this?

```
rack load-balancers node get --address 10.1.1.2
```

**Comment by jrperritt** [comment]
Yeah. Does an address uniquely identify a node associated with a given load balancer or is it an address/port combination? I'd guess you could send traffic to the same address but different ports, so using only address might return an error if there's more than one node with that address. Similar to how using a `name` flag to identify a resource may fail if the name isn't unique.

**Comment by jrperritt** [comment]
Of course, you'd need the ID of the LB:
rack load-balancers node get --load-balancer lbID --address 10.1.1.2

**Comment by rgbkrk** [comment]
Oh, it tends to be an address port combination.

**Comment by rgbkrk** [comment]
Should

```
rack load-balancers load-balancer create --name lb1 --nodes nodeID1,nodeID2 --port 80
```

be

```
rack load-balancers instance create --name lb1
```

Also, don't you have to have the load balancer before you can create the nodes (and by extension know the nodeID)?

**Comment by jrperritt** [comment]
Partially correct. It won't be the node IDs, but you can specify the nodes at LB creation time:
```
rack load-balancers load-balancer create --name lb1 --node address=192.0.2.1,port=80,type=primary,condition=enabled --node address=172.20.0.1,port=80,condition=disabled --port 80
```
